.editorconfig

```diff
@@ -9,6 +9,9 @@ end_of_line = lf
 charset = utf-8
 indent_style = space
 
+[Makefile]
+indent_style = tab
```

Comment:
brilliant! I'm always having this problem.

`/me adds this to his default .editorconfig`

🎉